### ISSUE TITLE
New pokedolls

### ISSUE THREAD

**Issue by Tommy06Pag** [Deltric/Generations#111]
Hi, it would be nice if these pokedolls were added, they come from the pokemon series (S2 ep1)
![image0](https://user-images.githubusercontent.com/81684661/116069353-90817400-a68b-11eb-9b10-512664500eb8.png)

**Comment by Deltric** [comment]
I don't believe we'd add these dolls as more so with dolls we're adding species and not really outfited Pokemon.
